### PR TITLE
CA branding

### PR DESCRIPTION
**Pull request by dmundra**
This merge request is to track customization like adding CA branding to the editor. We want to grab pieces of this to make it generic where possible.

### CODE REVIEW

**Comment by dmundra** [comment]
@maggiewachs making this PR a draft as we probably won't merge it as is but you can use to track your customizations.

File: src/components/report/ReportChapterTableResult.svelte
Comment (dmundra) [inline src/components/report/ReportChapterTableResult.svelte:62]: I couldn't help but notice that removing the label displays the conformance level with no context. Screenshot: 
Screenshot of a criteria example with conformance level missing web, electronic documents, software, authoring tool

Is there another display we should try to keep those labels? I wanted to avoid the table in a table look so open to ideas.

Comment (dmundra) [reply src/components/report/ReportChapterTableResult.svelte:62]: @maggiewachs ☝🏾

Comment (maggiewachs) [reply src/components/report/ReportChapterTableResult.svelte:62]: @dmundra I think the only issue here is that the labels need to be conditional. If we're only assessing web criteria, for example, we don't want to see "Web:" precede every conformance level and comment. But if you're assessing web and software, the labels are needed to differentiate. Does that make sense?

Comment (dmundra) [reply src/components/report/ReportChapterTableResult.svelte:62]: Ah great point. I will add that conditional check.

Comment (dmundra) [reply src/components/report/ReportChapterTableResult.svelte:62]: In https://github.com/CivicActions/openacr-editor/pull/13 I have the solution to hide labels if only one level is selected.

File: src/components/report/ReportChapterTableResult.svelte
Comment (mgifford) [inline src/components/report/ReportChapterTableResult.svelte:51]: So these are just commented out because??

File: src/components/report/ReportHTMLDownload.svelte
Comment (mgifford) [inline src/components/report/ReportHTMLDownload.svelte:37]: We had wanted to have an external style sheet, but if we did that the html document couldn't be a stand alone document (MHTML), right? 

Just trying to figure out why we're replacing so much CSS here.

File: src/components/report/ReportHeader.svelte
Comment (mgifford) [inline src/components/report/ReportHeader.svelte:10]: Should this be pushed upstream?

File: src/components/report/ReportHeader.svelte
Comment (mgifford) [inline src/components/report/ReportHeader.svelte:18]: More generally we could say it is "Based on: "

It isn't strictly following 2.4, but then generally they don't. Everyone modifies the template to do as they wish.

File: src/components/report/ReportHeader.svelte
Comment (mgifford) [inline src/components/report/ReportHeader.svelte:47]: We remove this because we're branding it, so it is clear?

File: src/components/report/ReportHeader.svelte
Comment (mgifford) [inline src/components/report/ReportHeader.svelte:65]: Worth pushing upstream?

File: src/components/report/ReportZipDownload.svelte
Comment (mgifford) [inline src/components/report/ReportZipDownload.svelte:101]: This looks like it should be pushed upstream.

File: src/components/report/ReportChapterTableResult.svelte
Comment (dmundra) [reply src/components/report/ReportChapterTableResult.svelte:51]: See comment https://github.com/CivicActions/openacr-editor/pull/1#discussion_r1104547904. I have a solution for it in the other merge request to allow it to be kept.

File: src/components/report/ReportHTMLDownload.svelte
Comment (dmundra) [reply src/components/report/ReportHTMLDownload.svelte:37]: Ya I created #11 to work on that but we might pivot to the CLI first and then come back to the editor.

File: src/components/report/ReportHeader.svelte
Comment (dmundra) [reply src/components/report/ReportHeader.svelte:10]: Yes in #13 I made the change.

File: src/components/report/ReportHeader.svelte
Comment (dmundra) [reply src/components/report/ReportHeader.svelte:18]: Ya #10 is the issue to tackle this one.

File: src/components/report/ReportHeader.svelte
Comment (dmundra) [reply src/components/report/ReportHeader.svelte:47]: Ya in #13 I provided a surrounding div to allow this to be hidden with CSS if needed.

File: src/components/report/ReportHeader.svelte
Comment (dmundra) [reply src/components/report/ReportHeader.svelte:65]: Yup in #13

File: src/components/report/ReportZipDownload.svelte
Comment (dmundra) [reply src/components/report/ReportZipDownload.svelte:101]: Potentially but if we allow custom CSS then we can leave the basic boilerplate and allow customization on top.

**Review by mgifford — APPROVED** [review]
Looks good to me.

File: src/components/report/ReportChapterTableResult.svelte
Comment (dmundra) [reply src/components/report/ReportChapterTableResult.svelte:51]: Solution in place and also in the CLI

File: src/components/report/ReportHTMLDownload.svelte
Comment (dmundra) [reply src/components/report/ReportHTMLDownload.svelte:37]: Even in the CLI using inline CSS to make it easier for the maintainer to update just one file. The reason for the editor including CSS is because I couldn't figure out earlier how to the style changes and I guess the zip file would allow additional of external CSS files. Edited the description of the issue.

**Comment by dmundra** [comment]
Closing this in favor of CLI changes: https://github.com/CivicActions/openacr/pull/5

**Pull request closed without merging**
